complex.c

```diff
@@ -1273,10 +1273,8 @@ rb_complex_finite_p(VALUE self)
  * call-seq:
  *    cmp.infinite?  ->  nil or 1
  *
- * Returns values corresponding to the value of +cmp+'s magnitude:
- *
- * +finite+::    +nil+
- * ++Infinity+:: ++1+
+ * Returns +1+ if +cmp+'s real and imaginary values are both infinite numbers,
```

Comment:
This is not true.
`Complex(0, Float::INFINITY).infinite?` returns `1`.

Reply:
You're right! It should be ``OR``

Reply:
@nobu right, should be ``OR``. Updated!